samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EventLogMessages_eu.properties

```diff
@@ -59,3 +59,8 @@ alt_sortErrorMsgDescending=eherantz ordenatu erroreekin
 t_sortIP=IP helbideekin ordenatu
 alt_sortIPAscending=Gorantz ordenatu IP helbideekin
 alt_sortIPDescending=Beherantz ordenatu IP helbideekin
+
+# S2U-24
```

Comment:
@pablosanroman Would you mind translating these strings?

Reply:
Hi @mpellicer 
export_csv=Esportatu CSV
datatables_entities=gertaerak